test/unit_test/engine/packaging/test_packaging_generator.py

```diff
@@ -341,7 +342,18 @@ def verify_output_artifacts(output_dir):
     assert (output_dir / "ONNXRuntimePackages").exists()
 
 
-def verify_models_rank_json_file(file_path):
+def verify_models_rank_json_file(output_dir, file_path, save_as_external_data=False, export_in_mlflow_format=False):
     with Path.open(file_path) as file:
         data = json.load(file)
+
     assert data is not None
+    # verify model path
+    for model_data in data:
+        model_path = output_dir / Path(model_data["model_config"]["config"]["model_path"])
+        assert model_path.exists(), "Model path in model rank file does not exist"
+        if export_in_mlflow_format:
+            assert mlflow.onnx.load_model(str(model_path)), "Model path in model rank file is not a valid MLflow model"
+        elif save_as_external_data:
+            assert onnx.load(str(model_path / "model.onnx")), "Model path in model rank file is not a valid ONNX model"
```

Comment:
The error messages are same no matter onnx model is saved with external data or not. 
Could you help update the message to distinguish that?

BTW it sounds mismatched between object and subject: `model path is not xxxxxx model`.

Reply:
Updated.